bootstrap_sdk

```diff
@@ -83,6 +83,54 @@ mkdir -p "${ROOT_OVERLAY}/tmp"
 chmod 1777 "${ROOT_OVERLAY}/tmp"
 cp "${BUILD_LIBRARY_DIR}/toolchain_util.sh" "${ROOT_OVERLAY}/tmp"
 
+
+# Stage 1 uses a different ebuild repo snapshot as well as a different portage_overlay 
+# the ones from the SDK where ./bootstrap_sdk is being executed -
+# to buils a known-good stage 1 (see stages description at top
+# of file). This prevents package upgrades from updated ebuild files in the portage/coreos
+# to apply in stage 1.
+# Stage 1 lacks proper isolation and will link all packages built against its own
+# seed libraries instead of against libraries installed to /tmp/stage1root.
+build_stage1() {
+    local stage1_repos="$TEMPDIR/stage1-ebuild-repos"
+
+    write_configs
+
+    # use known-good gentoo base repo from SDK, coreos-overlay from manifest
```

Comment:
Isn't this gentoo base repo being our portage-stable?

Reply:
Yes; it is generally referred to as the "gentoo base" repo in the SDK scripts (and also in the gentoo catalyst / bootstrap tooling).

Reply:
I don't understand the code... Why is coreos-overlay called out explicitly, but portage-base not?